### ISSUE TITLE
RL11 booting problem with PDP11/RT-11

### ISSUE THREAD

**Issue by kolloid** [simh/simh#412]
I cannot boot any version of RT-11 from a RL11 disk with SIMH 4.0 beta.
The error I get each time is:

`PDP-11 simulator V4.0-0 Beta        git commit id: b41d10f1`
`Disabling XQ`

`HALT instruction, PC: 000002 (HALT)`
`sim>`

Booting from RK disks works, and also booting from the same RL11 disk with the PDP11 emulator from SIMH 3.9, using the same init files.

This problem happens even from inside RT-11. If I boot from a RK disk, which works, and then use a RT-11 command to boot an attached RL11 disk, the emulator crashes.

I have attempted to follow the execution of the code a bit. The PDP-11 assembly boot code in the RL-11 handler gets executed, so the error happens ahead. Before spending further time I would like to check if someone else has experienced and (hopefully) corrected this. 

My environment is Linux Mint 64 bits based on Ubuntu 14.04, compiler is gcc 5.3.0. My version of SIMH is self-compiled. 
Thanks
Tom

**Comment by markpizz** [comment]
Please zip up the disk image you're trying to boot and drop it here.  Also provide the pdp11 configuration file you're running with.

**Comment by kolloid** [comment]
Here are both files:
[PDP11.zip](https://github.com/simh/simh/files/835168/PDP11.zip)

pdp11 is a 64 bit executable. However, just in case, I also compiled SIMH 4.0 in a 32 bit virtual Linux machine. The results were the same...
Thanks for your interest
Tom

**Comment by kolloid** [comment]
You asked for the disk, I sent you the executable, my bad, sorry.
Here goes again...
[RT11_disk.zip](https://github.com/simh/simh/files/835182/RT11_disk.zip)

Tom

**Comment by markpizz** [comment]
Tom,

The PDP11.zip file has a rt11.cnf which mentions a disk image called: TSX11_6.2_RL02.dsk, but the zip does not contain this file...

- Mark

**Comment by kolloid** [comment]
Yes, you are right. I sent a second post afterwards with the correct image. In any case, I'm sending yet another, which also fails. This is the standard distribution of RT11-v5.3.
[RT11-V5.3.zip](https://github.com/simh/simh/files/835197/RT11-V5.3.zip)

However, the issue is more complex that I thought at first: while RT11 v5.3 fails, an older distribution, RT11 v4 works. Hmmm...

Tom

**Comment by markpizz** [comment]
Can you provide the older disk image as well?

Thanks.

**Comment by kolloid** [comment]
Yes, I'm sending it, but I'm afraid that I spoke too soon, the situation here seems to be different.
I took the RT-11 distribution on RK and with RT-11 commands generated the same distribution on a RL disk. This didn't give any errors, so I booted the new RL disk with:
`.R DUP`
`*DL0:/O`


which worked and the RL disk booted. So I thought that everything worked. However now I was rechecking and what happens is that this RL disk does not boot from SIMH 4.0  and neither from SIMH 3.9 . I may have made some error generating this bootable disk, but again it boots from RT11, which I think is almost a hard boot, so for now I'm a bit stumped. It might be the same problem or not. I'm sending both disks, the RK and the RL, just in case ...

[RT11-V4_RL_RK.zip](https://github.com/simh/simh/files/835284/RT11-V4_RL_RK.zip)

Tom
